src/app/static/src/app/store/projects/actions.ts

```diff
@@ -85,9 +88,9 @@ export const actions: ActionTree<ProjectsState, RootState> & ProjectsActions = {
             .postAndReturn<string>("/project/", qs.stringify({name}));
     },
 
-    async getProjects(context) {
+    async getProjects(context, isLoading = true) {
         const {commit} = context;
-        commit({type: ProjectsMutations.SetLoading, payload: true});
+        commit({type: ProjectsMutations.SetLoading, payload: isLoading});
```

Comment:
This is fine, but I really meant to just not do the commit at all if the isLoading parameter is false - this achieves the desired behaviour, but is doing an unnecessary commit. Sorry, should have clarified that.